docs/userman/run-deploy/deploy/linux/redhat.md

```diff
@@ -67,11 +67,11 @@ taipy
 
 On your local machine, start the application by doing:
 ```console
-$ python3.8 -m pip install -r requirements.txt
+$ python3.9 -m pip install -r requirements.txt
```

Comment:
Why not just `python3` or even `python`?? Why bother?

Reply:
It's on the deployment doc for Redhat. We've already had a `info` block saying that you can or should replace `python3.9` with `python`